webdriver/tests/bidi/script/evaluate/exception_details.py

```diff
@@ -3,177 +3,18 @@
 
 from ... import any_int, any_string, recursive_compare
 from .. import any_stack_trace
+from . import PRIMITIVE_VALUES, REMOTE_VALUES
 
 
 @pytest.mark.asyncio
-async def test_invalid_script(bidi_session, top_context):
+@pytest.mark.parametrize("expression, expected", PRIMITIVE_VALUES + REMOTE_VALUES)
```

Comment:
this still looks like a lot of tests, I wonder if it would still fail sometimes.